test/resources/reftest-analyzer.js

```diff
@@ -225,7 +225,9 @@ window.onload = function () {
         });
         continue;
       }
-      match = line.match(/^ {2}IMAGE[^:]*\((\d+)x(\d+)x(\d+)\): (.*)$/);
+      match = line.match(
+        /^ {2}IMAGE[^:]*\((\d+\.?\d*)x(\d+\.?\d*)x(\d+\.?\d*)\): (.*)$/
```

Comment:
This will theoretically allow an outputScale that's e.g. just `1.` or similar, however I assume that's not really a problem given how that value is set in `test/driver.js`?

Reply:
The value is passed in `parseFloat` so it isn't a problem.
And if it is an integer it should just be `1`.